### PR TITLE
CMD-10-Ensure-execution-runs-correctly--options--arguments-

### PR DESCRIPTION
**Pull request by djon2003**

### CODE REVIEW

**Review by djon2003 — COMMENTED** [review]
Self-reviewed

File: CmdStarter.Tests/TestsCommon.cs [hunk 1]
```diff
@@ -116,30 +115,46 @@ public static void AssertIEnumerablesHaveSameElements<T>(IEnumerable<T> actual,
...
-                        return OptHandling.OPTION_PREFIX + optionName + " " + expectedValue;
+                        var value = expectedValue.ToString() ?? string.Empty;
+                        if (!string.IsNullOrWhiteSpace(value))
+                        {
+                            args.Add(OptHandling.OPTION_PREFIX + optionName);
+                            args.Add(value);
+                        }
                     }
+                    break;
             }
```
Comment (skydeszka) [inline CmdStarter.Tests/TestsCommon.cs:150]: Isn't the switch case unjustified here?

Switch cases are less readable than if statements for just two elements.

Comment (djon2003) [reply CmdStarter.Tests/TestsCommon.cs:150]: Well, it is a switch to support other types in the future. I know, for now, it is a bit unjustified.

Comment (skydeszka) [reply CmdStarter.Tests/TestsCommon.cs:150]: Oh, okay!

**Pull request merged**